docs/modules/index.md

```diff
@@ -0,0 +1,36 @@
+# Pre-configured Containers
```

Comment:
Are they already called and known by the community as `pre-configured containers`? We use the word `module` in general for this

Reply:
I do not like to call them modules yet. They are not really capable of any vendor specific features, etc. I would like to introduce "modules" with the dedicated packages.

Reply:
I would rather not introduce a new word for this in another language (`pre-configured containers` in this case). While we don't have a clear definition of a module for all languages (and not even an explicit definition within Testcontainers for Java), using `module` here already, should in general help with our messaging and creating consistency across languages, even if in .NET, the capabilities are still limited.

Looking at their [implementation](https://github.com/testcontainers/testcontainers-dotnet/tree/develop/src/Testcontainers/Containers/Modules), they are already quite aligned with our core idea of a container module (handling robust startup of a container).

Reply:
I changed it to Modules.